tests/drivers/pwm/pwm_api/boards/mimxrt1024_evk.overlay

```diff
@@ -0,0 +1,11 @@
+/ {
+	aliases {
+		pwm-0 = &flexpwm1_pwm2;
+    };
+};
+
+&flexpwm1_pwm2 {
+	pinctrl-0 = <&pinmux_flexpwm1>;
+    pinctrl-names = "default";
```

Comment:
Indent with tabs here

Reply:
Ahh yes, sorry about that...you know how text editors can be..should be fixed now.

Reply:
you must use emacs